core/query/index.go

```diff
@@ -89,6 +89,8 @@ func (ind *Indexer) insertAnnotatedTxs(ctx context.Context, b *bc.Block) ([]*Ann
 
 	// Build the fully annotated transactions.
 	for pos, tx := range b.Transactions {
+		hashes = append(hashes, tx.ID[:])
+		positions = append(positions, uint32(pos))
```

Comment:
I think these are also an artifact of the rebase. `hashes` and `positions` are populated down below in the `for pos, tx := range annotatedTxs {` loop